guides/WRITING-DOCUMENTATION.md

```diff
@@ -234,25 +208,23 @@ If a React component expects any `props`, you should document them within the co
 Button.propTypes = {
   label: React.PropTypes.node.isRequired,
   /**
-   * Overwrite the button's `class` attribute by providing your own
-   */
+  Overwrite the button's `class` attribute by providing your own
```

Comment:
I think React will choke on this formatting? -- it's weirdly specific sometimes, but I can't remember off the top of my head if it requires that `*` in the comment block